### ISSUE TITLE
Refining the tutorial documentation

### ISSUE THREAD

**Issue by haifeng-jin** [keras-team/autokeras#408]
Read and try out the examples and tutorials in "[Getting Started](https://autokeras.com/tutorial/overview/)".
If you find anything unclear or incorrect, please change it in [start.md](https://github.com/keras-team/autokeras/tree/master/docs/templates/tutorial).

External contributors are welcome for solving this issue.

- [ ] Resume training.
- [ ] ImageRegressor
- [ ] TextRegressor
- [ ] StructuredDataRegressor
- [ ] customize metric and loss

**Comment by francotheengineer** [comment]
@jhfjhfj1  Hi, maybe some of the more basic tutorials can be made into Jupyter Notebooks? - I'm happy to write some in the style TF does (https://www.tensorflow.org/hub/tutorials/image_retraining) Thanks

**Comment by jhfjhfj1** [comment]
@francotheengineer Yes, we are happy to see them made into Jupyter Notebooks. We would appreciate if you can help us do it. Thanks.

**Comment by danucalovj** [comment]
Source link for Sentiment Analysis in the documentation is broken. Points to:

https://github.com/keras-team/autokeras/blob/master/autokeras/pretrained/text_classifier.py

404 Error

**Comment by kalufinnle** [comment]
the link of bayesian searcher is broken under Documentation

**Comment by crypdick** [comment]
The documentation for exporting models is very poor.
